pkg/drivers/kic/oci/types.go

```diff
@@ -56,6 +57,8 @@ type CreateParams struct {
 	Envs          map[string]string // environment variables to pass to the container
 	ExtraArgs     []string          // a list of any extra option to pass to oci binary during creation time, for example --expose 8080...
 	OCIBinary     string            // docker or podman
+	Network       string            // network to use for the containe
```

Comment:
Can you make this a clearer comment, with perhaps an example value? I don't know whether to expect a name or a CIDR here.